Experiemental Docker with mongo support broken

From https://hub.docker.com/r/moveit/moveit/builds/bybu65qy9524f3cazmfftxm/

Something is wrong with the Docker build setup by @ugocupcic

```
Errors << warehouse_ros_mongo:cmake /root/ws_moveit/logs/warehouse_ros_mongo/build.cmake.000.log
CMake Error at /opt/ros/jade/share/catkin/cmake/catkin_package.cmake:291 (message):
  catkin_package() absolute include dir
  '/root/ws_moveit/devel/.private/warehouse_ros_mongo/include' does not exist
Call Stack (most recent call first):
  /opt/ros/jade/share/catkin/cmake/catkin_package.cmake:98 (_catkin_package)
  CMakeLists.txt:24 (catkin_package)


cd /root/ws_moveit/build/warehouse_ros_mongo; catkin build --get-env warehouse_ros_mongo | catkin env -si  /usr/bin/cmake /root/ws_moveit/src/warehouse_ros_mongo --no-warn-unused-cli -DCATKIN_DEVEL_PREFIX=/root/ws_moveit/devel/.private/warehouse_ros_mongo -DCMAKE_INSTALL_PREFIX=/root/ws_moveit/install; cd -
```

I will likely disable this build if no one is interested in fixing it, to stop the emails I'm getting.

I'll try to take a look in the next few days 

Sent from a tiny keyboard, excuse the brevity.

> On 18 Oct 2016, at 19:24, Dave Coleman notifications@github.com wrote:
> 
> From https://hub.docker.com/r/moveit/moveit/builds/bybu65qy9524f3cazmfftxm/
> 
> Something is wrong with the Docker build setup by @ugocupcic
> 
> Errors << warehouse_ros_mongo:cmake /root/ws_moveit/logs/warehouse_ros_mongo/build.cmake.000.log
> CMake Error at /opt/ros/jade/share/catkin/cmake/catkin_package.cmake:291 (message):
>   catkin_package() absolute include dir
>   '/root/ws_moveit/devel/.private/warehouse_ros_mongo/include' does not exist
> Call Stack (most recent call first):
>   /opt/ros/jade/share/catkin/cmake/catkin_package.cmake:98 (_catkin_package)
>   CMakeLists.txt:24 (catkin_package)
> 
> cd /root/ws_moveit/build/warehouse_ros_mongo; catkin build --get-env warehouse_ros_mongo | catkin env -si  /usr/bin/cmake /root/ws_moveit/src/warehouse_ros_mongo --no-warn-unused-cli -DCATKIN_DEVEL_PREFIX=/root/ws_moveit/devel/.private/warehouse_ros_mongo -DCMAKE_INSTALL_PREFIX=/root/ws_moveit/install; cd -
> I will likely disable this build if no one is interested in fixing it, to stop the emails I'm getting.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

## 

## _We're hiring Software Developers! - visit our website for more details._

_Shadow Robot Company Ltd._
251 Liverpool Road, N1 1LX, UK

## Registered Number 3308007 (England & Wales)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,6 +19,8 @@ if("${MongoDB_LIBRARIES}" MATCHES "\\.so$")
   set(MONGO_EXPORT MongoDB)
 endif()
 
+file(MAKE_DIRECTORY "${CATKIN_DEVEL_PREFIX}/include")
+
 catkin_python_setup()
 
 catkin_package(
@@ -32,7 +34,6 @@ if (NOT MongoDB_EXPOSE_MACROS)
   add_definitions(-DMONGO_EXPOSE_MACROS)
 endif()
 
-file(MAKE_DIRECTORY "${CATKIN_DEVEL_PREFIX}/include")
 configure_file("include/warehouse_ros_mongo/config.h.in" "${CATKIN_DEVEL_PREFIX}/include/warehouse_ros_mongo/config.h")
 
 include_directories(${CATKIN_DEVEL_PREFIX}/include include ${catkin_INCLUDE_DIRS} ${MongoDB_INCLUDE_DIR} ${Boost_INCLUDE_DIRS})

--- a/package.xml
+++ b/package.xml
@@ -19,7 +19,6 @@
   <build_depend>rostime</build_depend>
   <build_depend>rostest</build_depend>
   <build_depend>std_msgs</build_depend>
-  <build_depend>mongodb-dev</build_depend>
   <build_depend>python-pymongo</build_depend>
   <build_depend>curl</build_depend>
   <build_depend>libssl-dev</build_depend>
@@ -28,11 +27,10 @@
   <run_depend>roscpp</run_depend>
   <run_depend>rostime</run_depend>
   <run_depend>std_msgs</run_depend>
-  <run_depend>mongodb-dev</run_depend>
   <run_depend>python-pymongo</run_depend>
   <run_depend>curl</run_depend>
   <run_depend>libssl-dev</run_depend>
-  
+
   <test_depend>gtest</test_depend>
 
   <export>

--- a/package.xml
+++ b/package.xml
@@ -30,6 +30,7 @@
   <run_depend>python-pymongo</run_depend>
   <run_depend>curl</run_depend>
   <run_depend>libssl-dev</run_depend>
+  <run_depend>mongodb</run_depend>
 
   <test_depend>gtest</test_depend>
 

